doc/nrf/protocols/matter/end_product/test_event_triggers.rst

```diff
@@ -53,17 +53,33 @@ The following table lists the available triggers and their activation codes:
   * - Factory reset
     - None
     - Perform a factory reset of the device with a delay.
-    - `0xF000000000000000` - `0xF00000000000FFFF`
-    - The range of `0x0000` - `0xFFFF` represents the delay in ms to wait until the factory reset occurs.
+    - `0xF000000000000000` - `0xF0000000FFFFFFFF`
+    - The range of `0x00000000` - `0xFFFFFFFF` represents the delay in ms to wait until the factory reset occurs.
       The maximum time delay is UINT32_MAX ms.
       The value is provided in HEX format.
   * - Reboot
     - None
     - Reboot the device.
-    - `0xF000000000000000` - `0xF00000000000FFFF`
-    - The range of `0x0000` - `0xFFFF` represents the delay in ms to wait until the reboot occurs.
+    - `0xF000000100000000` - `0xF0000001FFFFFFFF`
```

Comment:
I think we're not allowed to use these IDs for non-standard test even triggers. Per spec:
> Values of EventTrigger in the range 0xFFFF_FFFF_0000_0000 through 0xFFFF_FFFF_FFFF_FFFF are
reserved for testing use by manufacturers and will not appear in CSA certification test literature.

Reply:
Oh, right, I didn't notice that. So we need to change also the existing ones to meet the requirements.